.cirrus.yml

```diff
@@ -166,3 +166,41 @@ static_binary_task:
     save_cache_script: |
         mv /nix .cache
         chown -Rf $(whoami) .cache
+
+# Build the static binary for arm64
+static_binary_arm64_task:
+    depends_on:
```

Comment:
If this task fails, it will probably happen for some critical "must merge" PR, and while you or I aren't around. 
 Would you mind adding an "escape hatch" comment [similar to what we have in podman](https://github.com/containers/podman/blob/6a6e86829071d20a7f7fc30d7346c95a6fd3927a/.cirrus.yml#L330-L332)?

Reply:
Yes, I'll added this as suggested.